lib/coarse_op.cu

```diff
@@ -124,6 +124,14 @@ namespace quda {
       calculateY<Float,vFloat,fineColor,fineSpin,24,coarseSpin>(Y, X, Yatomic, Xatomic, uv, av, T, g, c, kappa, mu, mu_factor, dirac, matpc);
     } else if (coarseColor == 32) {
       calculateY<Float,vFloat,fineColor,fineSpin,32,coarseSpin>(Y, X, Yatomic, Xatomic, uv, av, T, g, c, kappa, mu, mu_factor, dirac, matpc);
+#if 0 // only needed for Laplace MG. Not necessary for staggered MG Lanczos
```

Comment:
Random comment: at some point we should fill in the missing pieces for regular Laplace MG!

Reply:
Agreed! The main change we'd, in principle, want to make is getting rid of preserving chirality, since it's meaningless. (That said, if we kept staggered chirality around on a first pass, all of the pieces may already be there.)